multiples.py

```diff
@@ -15,6 +16,16 @@ def good(i):
     return by_3 or by_5
 
 
+def even_elements_mult(i):
+    if i == 0:
+        return False
+    for num in i:
+        if num % 2 == 0 and num > 0:
+            print (num)
+        result = num * num
```

Comment:
There are multiple problems:
1) you define result inside the `for` block (I am surprised it does work, actually) - compare it with the sum in the previous example - we define it at the beginning and add TO IT  all the way. Here for every for loop iteration, you define a variable named `result` and assign it to num ^ 2. So when you finish you are returning the last value of the result which was 9*9 (last number in the loop)
2) Identation. Line `result = num * num` is not in the if block,  it is on the same level, meaning you will not filter odd numbers and you will call this line for EVERY number in the loop
3) I am not sure if this is valid syntax for the loop `for num in another_num:`, surprised this does work, you are supposed to use range(i) or at least explain what is that syntax you are using.

Reply:
please have a look on the latest changes

Reply:
https://github.com/irinaBaB/python_exercise/commit/4c64bc2ad1e43ff3386b24d8b1206d739c32fc24